src/constants.js

```diff
@@ -1,2 +1,3 @@
 export const PAGE = document.getElementById('main');
+export const API_URL = 'http://local/api';
```

Comment:
Why did you create this const `API_URL`?

Reply:
I really have no idea 😅 
I'm going to remove.